test/webpack.config.js

```diff
@@ -102,21 +116,34 @@ function getFirstKey(object) {
 
 module.exports = env => {
   env = env || {};
-  if (env.bench) {
-    return BENCH_CONFIG;
-  }
-  if (env.test) {
-    return TEST_CONFIG;
+
+  let config = COMMON_CONFIG;
```

Comment:
I found that having this switch case statement resulted in much more code noise than simple returns and avoids to have to reassign config, but that's a personal preference I guess